utils/genesis/src/lib.rs

```diff
@@ -103,12 +103,9 @@ where
             )
         })?;
 
-    let store_genesis = chain_store.genesis()?;
+    let store_genesis = chain_store.genesis();
 
-    if store_genesis
-        .map(|store| store == genesis_block)
-        .unwrap_or_default()
-    {
+    if store_genesis.is_ok() && store_genesis.unwrap() == genesis_block {
         debug!("Genesis from config matches Genesis from store");
```

Comment:
This debug message isn't important. Let's refactor the code to always set the genesis.

Reply:
Removed the message